extensions/sql-database-projects/src/models/project.ts

```diff
@@ -603,10 +603,10 @@ export class Project implements ISqlProject {
 		let result: ResultStatus;
 
 		if (path.extname(filePath) === constants.sqlFileExtension) {
-			result = await this.sqlProjService.addSqlObjectScript(this.projectFilePath, filePath)
```

Comment:
this was passing the full filepath, but it should pass the relative path to the .sqlproj